syft/cataloging/filecataloging/config.go

```diff
@@ -72,7 +72,12 @@ func (cfg Config) WithSelection(selection file.Selection) Config {
 	return cfg
 }
 
-func (cfg Config) WithHashers(hashers []crypto.Hash) Config {
+func (cfg Config) WithHashers(hashers ...crypto.Hash) Config {
 	cfg.Hashers = intFile.NormalizeHashes(hashers)
 	return cfg
 }
+
+func (cfg Config) WithContent(content filecontent.Config) Config {
```

Comment:
I don't understand this method. What is it for, and why is it part of this PR?

Reply:
in an earlier commit there was a refactor comment highlighting this was missing and difficult to use in the examples.  Since then I didn't end up using it in the example, but added it since I also had the comment that the crypto.Hash list should be variadic in the same PR (which was only a few lines over from this change).